packages/fxa-event-broker/lib/selfUpdatingService/clientCapabilityService.ts

```diff
@@ -35,7 +36,9 @@ class ClientCapabilityService extends SelfUpdatingService<ClientCapabilities> {
   private readonly clientUrl: string;
 
   constructor(logger: Logger, config: ServiceConfig) {
-    super(logger, config.refreshInterval * 1000, {});
+    const requireCapabilities =
+      config.requireCapabilities !== undefined ? config.requireCapabilities : true;
```

Comment:
Would it ever actually be `undefined`? I thought the whole point of defaults in convict was that it takes care of giving you the default value in the fallback case, there's no need to implement that code yourself.

Reply:
I wondered the same to be honest, and then immediately forgot to comment on it.

Reply:
The code can't know the entire calling chain. It is possible to create this object with an undefined requireCapabilities per its ServiceConfig interface, so this has to be checked for type safety.